libs/features/georoutingv2/georoutingv2.js

```diff
@@ -110,14 +111,18 @@ function getGeoroutingOverride() {
   return hideGeorouting === 'on';
 }
 
-function decorateForOnLinkClick(link, prefix) {
+function decorateForOnLinkClick(link, prefix, currPrefix) {
```

Comment:
I think naming here is a bit confusing. Looking at a call of this method, ` decorateForOnLinkClick(altAction, currentPage.prefix, locale.prefix);`, `currentPage.prefix` corresponds to `prefix`, while `locale.prefix` corresponds to `currPrefix`. If I isolated this method, I would have no idea what the difference between `prefix` and `currPrefix` is. Maybe some different naming can be found. `urlLocale` and `pageLocale`? `pagePrefix` and `urlPrefix`? Something along those lines, you probably know the use-case better 😀

Reply:
this is fixed